src/modules/flow/iio/nodes.c

```diff
@@ -49,16 +49,16 @@ reader_cb(void *data, struct sol_iio_device *device)
         .min = mdata->out_range.min,
         .max = mdata->out_range.max
     };
-    bool b;
+    int b;
```

Comment:
use r instead of b here. People that are reading the code may think it is a boolean because it is called b.